timescaledb/load_data.py

```diff
@@ -97,10 +101,9 @@ def read_from_summary(path):
         terminals (str): The number of terminals used for benchmark testing.
 
     """
-    f = open((BASEPATH / path).resolve())
+    f = open(path)
```

Comment:
If you use `with open()` you don't need to explicitly call `f.close()` https://stackoverflow.com/questions/31334061/file-read-using-open-vs-with-open

Reply:
Done.